examples/sp/params.txt

```diff
@@ -0,0 +1,46 @@
+# PRIO PARAMETERS
```

Comment:
Since it is possible to insert comment lines in this file, wouldn't it make sense to include a header at the beginning very shortly describing what sort of computation this is? At least the computational protocol and molecular system name could be given so that the user does not need to look for this info below or in other files.

Reply:
Will do, I'll adjust the file anyway. Do you think it's better to keep all parameters for this calculation in the file or should I remove the parameters that will be given by default?

Reply:
I believe, this is an input example intended to show new coming users how to run computations with `gmx2qmmm`.  In this regard, we should not over complicate it with technical parameters, which are set to default values anyway. But I should also mention that some program packages have a default value for the exchange-correlation functional. However, it is weird not to put it into an input example, since the user should know how to change this setting.

So, the conclusion is to make it as simple as possible, but still keep important things in :-)